pkg/chat/handler.go

```diff
@@ -128,7 +144,7 @@ func (h *ChatHandler) GetCatagorysByMessageID(c echo.Context) error {
 // @Param catagoryID path uint true "catagory ID"
 // @Success 200 {object} api.ReturnedData{data=[]string}
 // @Router /keyword/{keywordID}/catagory/{catagoryID}  [GET]
-func (h *ChatHandler) GetQuestiosByCatagoryID(c echo.Context) error {
+func (h *KeywordHandler) GetQuestionsByCatagoryID(c echo.Context) error {
```

Comment:
Why do we need keywordID here?

Reply:
Emmm...do you mean that the router is wrong or the keywordhandler is unnecessary?

Reply:
Well that's ok